twoliter/src/cmd/make.rs

```diff
@@ -62,8 +63,10 @@ impl Make {
             .env("CARGO_HOME", self.cargo_home.display().to_string())
             .env("TWOLITER_TOOLS_DIR", toolsdir.display().to_string())
             .env("BUILDSYS_VERSION_IMAGE", project.release_version())
+            .env("TWOLITER_QUIET", if global_opts.quiet { "1" } else { "0" })
             .makefile(makefile_path)
             .project_dir(project.project_dir())
+            .quiet(global_opts.quiet)
```

Comment:
Q - looking at the code, it seems that the `CargoMake.quiet` field is not really used? Only the env var was used.